aten/src/ATen/native/MaxPooling.cpp

```diff
@@ -97,6 +97,10 @@ Tensor max_pool1d(
     IntArrayRef padding,
     IntArrayRef dilation,
     bool ceil_mode) {
+  if (self.is_quantized()) {
```

Comment:
I was wondering if we can either delete this or delete the `quantized_max_pool1d`?

Reply:
That's a good point -- I'll try

Reply:
Actually, I started getting weird errors about the derivative -- I will factor it out in later PRs, if you don't mind